test/utils/utils_linux.cpp

```diff
@@ -56,3 +56,17 @@ TEST_F(test, utils_shm_create_invalid_args) {
     ret = utils_shm_create("/abc", -1);
     EXPECT_EQ(ret, -1);
 }
+
+TEST_F(test, utils_get_size_threshold) {
+    // Expected input to utils_get_size_threshold():
+    // char *str_threshold = utils_env_var_get_str("UMF_PROXY", "size.threshold=");
+    // positive tests
+    EXPECT_EQ(utils_get_size_threshold((char *)"size.threshold=111"), 111);
+    EXPECT_EQ(utils_get_size_threshold((char *)"size.threshold=222;abcd"), 222);
+    EXPECT_EQ(utils_get_size_threshold((char *)"size.threshold=333;var=value"),
```

Comment:
I'm missing positive test case with precedding variable (e.g. `"var1=value1;size.threshold=333;var2=value2"`)

Reply:
```c
    // Expected input to utils_get_size_threshold():
    // char *str_threshold = utils_env_var_get_str("UMF_PROXY", "size.threshold=");
```
always begins with `"size.threshold="`.

Reply:
It is tested by this test itself:
```
        set_property(TEST umf-test_proxy_lib_size_threshold
                     PROPERTY ENVIRONMENT UMF_PROXY="page.disposition=shared-shm;size.threshold=64")
```

Done